### PR TITLE
chromium@124.0.6367.90: Fix hash

### LINKED ISSUE #162: chromium@124.0.6367.90: hash check failed
Checking hash of ungoogled-chromium-124.0.6367.90-1_Win64.7z ... ERROR Hash check failed!
App:         business/chromium
URL:         https://github.com/macchrome/winchrome/releases/download/v124.6367.90-M124.0.6367.90-r1274542-Win64/ungoogled-chromium-124.0.6367.90-1_Win64.7z
First bytes: 37 7A BC AF 27 1C 00 04
Expected:    28d97ca3088e7de6a994df5c7e1e3b119b679a32
Actual:      35cfb396dc50b98e53f99016076ac549fbc50e52

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #162

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### chromium

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
